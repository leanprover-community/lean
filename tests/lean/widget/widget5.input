{"seq_num": 0, "command": "sync", "file_name": "widget5.lean"}
{"seq_num": 45, "command": "info", "file_name": "widget5.lean", "line": 3, "column": 3}
{"seq_num": 1, "command": "get_widget", "file_name": "widget5.lean", "column":0,"id":15,"line":3}
