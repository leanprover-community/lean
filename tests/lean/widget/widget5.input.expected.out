{"msgs":[{"caption":"trace output","file_name":"widget5.lean","pos_col":0,"pos_line":3,"severity":"information","text":"successfully rendered widget\n"}],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"widget":{"column":0,"id":15,"line":3}},"response":"ok","seq_num":45}
{"response":"ok","seq_num":1,"widget":{"column":0,"html":{"c":[{"a":{"style":{"color":"blue"}},"c":["hi"],"t":"p"}]},"id":15,"line":3}}
